release/k3s/k3s.go

```diff
@@ -18,6 +18,7 @@ import (
 	"github.com/go-git/go-git/v5/plumbing"
 	"github.com/go-git/go-git/v5/plumbing/transport/ssh"
 	"github.com/google/go-github/v39/github"
+	ecmExec "github.com/rancher/ecm-distro-tools/exec"
```

Comment:
We can leave the import reference `exec` here since it's not mirroring a stdlib import.